### PR TITLE
Restaurant Page: Javascript 

### PR DESCRIPTION
**Pull request by Stephanie041996**
-In this project, I dynamically rendered a simple restaurant homepage using JavaScript alone to generate the entire contents of the website.
- I used Webpack and modules to create this project.

### CODE REVIEW

**Review by mahmoud717 — CHANGES_REQUESTED** [review]
### Changes required ♻️ 

**Hello @Stephanie041996**

Great job with the project so far it looks amazing, but there are still some changes to be made:
 
---
### Requested 🔴 

- [x] Please make sure to remove any commented code. 
- [x] Make sure to add your images in their own folder instead of leaving them hovering all over the /src directory. I would suggest naming that folder /assets.
- [x] your PR should have a descriptive name and description, that would indicate what have you done in this project.


---

please make these changes and request another code review.

if you have any questions don't hesitate to contact me on slack **MahmoudMohammad**

**Comment by Stephanie041996** [comment]
thank you for the review, I have made the necessary changes

File: package.json [hunk 1]
```diff
@@ -4,7 +4,6 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
     "build": "webpack"
   },
```
Comment (SajjadAhmad14) [inline package.json]: Please add this to scripts inside package.json file:
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

### These changes are must to make before merging branch with development.
- [x] Please add `webpack-dev-server` in package.json file. Kindly see this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/devdep_requirement.png).
- [x] Scripts inside package.json file should be as this:
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
- [x] Please see inline comments for reference.
### Happy Coding ☺️

## If you have any questions you can comment here in this PR or can always connect with me on slack ( @Sajjad Ahmad ).

**Pull request merged**